internal/expr/parser.go

```diff
@@ -257,37 +249,36 @@ func (p *Parser) skipComment() bool {
 	return false
 }
 
-// Parse takes an input string and parses the input and output parts. It returns
-// a pointer to a ParsedExpr.
-func (p *Parser) Parse(input string) (expr *ParsedExpr, err error) {
+// Parse takes an SQLair query string and returns a ParsedExpr.
+func (p *Parser) Parse(query string) (pe *ParsedExpr, err error) {
```

Comment:
I think query makes sense in regard to this being a public method, we want to say what the argument is to the user, not to us. But since this is in the internal package it makes sense to change it back to input.